### DISCUSSION TITLE
Elastic agent in SO.

### DISCUSSION THREAD

**Discussion by Jackson-Pollock** [Security-Onion-Solutions/securityonion#5831] [General] [answered]
Just wanted to check if SO is planning the support of ELK SIEM along with Elastic Agent in future.

**Comment by dougburks** [comment, marked as answer]
Yes, we plan to support Elastic Agent in the future.

**Reply by EchoGangster** [reply]
That is great news.  Is there a release you are aiming to have this implemented?

**Reply by dougburks** [reply]
Elastic Agent currently sends directly to Elasticsearch and thus requires opening the Elasticsearch port to your endpoints. They are working on a new feature to allow Elastic Agent to send to Logstash instead. We are waiting on that since that allows for a much smaller attack surface.

**Reply by 8bitjoe** [reply]
any news on this? i see that logstash output is already a thing

**Reply by EchoGangster** [reply]
I think this is getting closer. There is now a logstash integration for the Elastic Agent in Technical Preview.

**Reply by Xboarder56** [reply]
They mentioned some of it here https://blog.securityonion.net/2023/02/security-onion-in-2022-and-2023.html

Probably will be here sometime this year for the 2.4 drop.
